### PR TITLE
feat: assessments

### PR DESCRIPTION
**Pull request by tenstad**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    DatasetsRequest:
+      type: object
+      properties:
+        datasets:
+          type: array
+          items:
+            type: string
+      required:
+        - datasets
+    ScorePutRequest:
```
Comment (jeffreiffers) [inline openapi.yaml:30]:
```suggestion
    ScorePostRequest:
```

File: src/db_models.rs [hunk 2]
```diff
@@ -2,17 +2,19 @@ use super::schema::*;
 use diesel::sql_types::Double;
 
 #[derive(Insertable, Queryable, AsChangeset)]
-#[table_name = "datasets"]
-pub struct Dataset {
+#[table_name = "dataset_assessments"]
+pub struct DatasetAssessment {
     pub id: String,
```
Comment (jeffreiffers) [inline src/db_models.rs:7]: Antar vi legger til timestamp i annen PR.

Comment (tenstad) [reply src/db_models.rs:7]: Timestamp ligger i assessment grafen.

File: openapi.yaml (hunk 1 above)
Comment (tenstad) [reply openapi.yaml]: Fixed https://github.com/Informasjonsforvaltning/fdk-mqa-scoring-api/pull/25/commits/d3bab006623f3c66f78f76d45aead499cf23b893

File: openapi.yaml [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+            type: string
+          required: true
+          description: Assessment UUID
+      requestBody:
+        description: Assessment in turtle and jsonld format, along with json scores
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ScorePutRequest"
```
[the remark this answers is not in the document]
Comment (tenstad) [reply openapi.yaml]: Fixed https://github.com/Informasjonsforvaltning/fdk-mqa-scoring-api/pull/25/commits/d3bab006623f3c66f78f76d45aead499cf23b893

**Pull request merged**